examples/nvflare/v2.2/mimic-cxr-nvflare/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# !! EDIT THIS: Set base Docker image to be used.
```

Comment:
This folder is missing a README.md to explain about this example, how to use it, etc.